test/readmeexamples.js

```diff
@@ -54,7 +54,7 @@ describe('#db README Examples', function() {
         .put(`/${DBNAME}`)
         .reply(201, { ok: true });
 
-      var cloudant = Cloudant({ username: ME, password: PASSWORD, url: SERVER });
+      var cloudant = Cloudant({ username: ME, password: PASSWORD, url: SERVER, plugins: [] });
```

Comment:
Similarly to above, but I think in this case the tests are required to run against the service. This is going to force them to use basic auth, is that the intent?

Reply:
I think we have to have the session mocks in this case, because the readmeexamples are definitely ones we want to run against the real service.